src/model-preparation.jl

```diff
@@ -315,7 +306,6 @@ function add_expression_terms_over_clustered_year_constraints!(
                 ARRAY_AGG(COALESCE(var.id, []) ORDER BY var.rep_period) AS var_id_vec,
                 ARRAY_AGG(COALESCE(var.time_block_start, []) ORDER BY var.rep_period) AS var_time_block_start_vec,
                 ARRAY_AGG(COALESCE(var.time_block_end, []) ORDER BY var.rep_period) AS var_time_block_end_vec,
-                ARRAY_AGG(COALESCE(var.efficiency, []) ORDER BY var.rep_period) AS var_efficiencies,
```

Comment:
This is another candidate for making it faster. Maybe it was overshadowed by other things making it slower?
If I'm not mistaken, this is part of the `create_model` part of the code.